opennmt/experiments/runs/multi_train_a100/config.yaml

```diff
@@ -0,0 +1,221 @@
+seed: 1234
+
+test_src: /dir/test.hu
+test_tgt: /dir/test.en
```

Comment:
not sure if this is a problem or this is just a sample config file that is edited manually, but just pointing out that we have `hu-en` language codes here, while below there are references to an `en-de` data directory.

Reply:
It's just a sample, but I fixed it